README.md

```diff
@@ -1,2 +1,42 @@
-# pyodide-blog
-The Pyodide blog  
+# Pyodide blog
+
+The contents of the Pyodide blog
+
+## Build locally
+
+To build locally clone this repository, and update submodules
+```
+cd pyodide-blog
+git submodule update --init --recursive
+```
+then build the blog with Hugo,
+```
+hugo server -D
+```
+
+Note: the Linux Hugo executable is included in this repo. If you are on Linux,
+change all `hugo` commands by `./hugo`. Otherwise if you are on a different OS,
```

Comment:
```suggestion
change all `hugo` commands to `./hugo`. Otherwise if you are on a different OS,
```

Reply:
Is it really a good idea to include the linux binary in the repo? Why not just tell people to `sudo apt-get install hugo`?

Reply:
Oh:
>This option is not recommended because the Hugo in Linux package managers for Debian and Ubuntu is usually a few versions behind as described here

Reply:
Yes, since apt is not recommended, the solution is to go to their Github release and download the .tar.gz with the binary. Maybe I should have just said that in the README. Also since this was originally a personal repo that I moved here, this PR is not made in the branch of a fork, the file is going to stay in git history even if I remove it now I believe. Unless one starts rewriting history with git filter-branch. 

Anyway I agree that it's not ideal, but it's 40MB not that critical either.